homeassistant/components/sensor/pvoutput.py

```diff
@@ -0,0 +1,115 @@
+"""
+Support for getting collected information from PVOutput.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.pvoutput/
+"""
+import logging
+from collections import namedtuple
+
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.components.sensor.rest import RestData
+from homeassistant.const import (
+    ATTR_TEMPERATURE, CONF_API_KEY, CONF_NAME, STATE_UNKNOWN)
+
+_LOGGER = logging.getLogger(__name__)
+_ENDPOINT = 'http://pvoutput.org/service/r2/getstatus.jsp'
+
+ATTR_DATE = 'date'
+ATTR_TIME = 'time'
+ATTR_ENERGY_GENERATION = 'energy_generation'
+ATTR_POWER_GENERATION = 'power_generation'
+ATTR_ENERGY_CONSUMPTION = 'energy_consumption'
+ATTR_POWER_CONSUMPTION = 'power_consumption'
+ATTR_EFFICIENCY = 'efficiency'
+ATTR_VOLTAGE = 'voltage'
+
+CONF_SYSTEM_ID = 'system_id'
+
+DEFAULT_NAME = 'PVOutput'
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_API_KEY): cv.string,
+    vol.Required(CONF_SYSTEM_ID): cv.string,
+    vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+})
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Set up the PVOutput sensor."""
+    name = config.get(CONF_NAME)
+    api_key = config.get(CONF_API_KEY)
+    system_id = config.get(CONF_SYSTEM_ID)
+    method = 'GET'
+    payload = verify_ssl = auth = None
```

Comment:
If a web service is HTTP-only we should still set verify SSL to True. It doesn't mean that it will break on HTTP. We should pretty much never touch that setting, it's dangerous and should not even touch it when we don't need it.